src/style.css

```
@@ -0,0 +1,75 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 300px;
+  color: #fff;
+}
+
+.block--1 {
+  background-color: red;
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background-color: black;
+}
+
+@media (min-width: 600px) {
+  .block--3 {
+    order: 1;
+  }
+  .block--5 {
+    order: 2;
+  }
+  .block--6 {
+    order: 3;
+  }
```

add newlines to improve readability
```suggestion
  .block--3 {
    order: 1;
  }
  
  .block--5 {
    order: 2;
  }
  
  .block--6 {
    order: 3;
  }
```